src/element.ts

```diff
@@ -491,4 +483,95 @@ export abstract class Element {
     const fontInfo = this.fontInfo;
     this.setFont(fontInfo.family, fontInfo.size, weight, fontInfo.style);
   }
+
+  /** Get element width. */
+  getWidth(): number {
+    return this.width;
+  }
+
+  /** Set element width. */
+  setWidth(width: number): this {
+    this.width = width;
+    return this;
+  }
+
+  /** Shift element down `y` pixels. Negative values shift up. */
+  setYShift(y: number): this {
```

Comment:
Should the parameter name be `dY` or `deltaY`? (or I suppose it could also be `yShift`)?

Reply:
done